src/api/app/assets/javascripts/webui/content-selector-filters.js

```diff
@@ -27,7 +27,8 @@ function submitFilters() {
 }
 
 let submitFiltersTimeout;
-$(document).on('change keyup', '#content-selector-filters-form input, #content-selector-filters-form select', function() {
+const autoSubmitOnChangeSelector = '#content-selector-filters-form .auto-submit-on-change';
```

Comment:
```suggestion
const autoSubmitOnChangeSelector = '.auto-submit-on-change';
```
Now that we introduce the `.auto-submit-on-change` class, we don't need to make it explicit that it belongs to the `#content-selector-filters-form` element. Do we?

Reply:
From a high level POV in the page there could be other places where an input may get assigned to that class, so we should prevent anything outside of the request filter to submit the filter form.
On the other hand, I agree the chance that we will end up in such a scenario is very remote, almost impossible.
But this is a specific scoped event implementation, and by default it is always better to stick to the narrowest and precise description of the element who subscribes to.
I'd rather stick to the `id` prefix if you don't mind.

Reply:
Hm... It doesn't make sense to narrow the scope of the selector if we currently don't need it... I'd prefer to remove it.

Reply:
> Hm... It doesn't make sense to narrow the scope of the selector if we currently don't need it... I'd prefer to remove it.

I see. We barely will ever have the conflict in place so, we can drop it this time.